library/core/src/marker.rs

```diff
@@ -540,7 +552,7 @@ macro_rules! impls {
 /// For a more in-depth explanation of how to use `PhantomData<T>`, please see
 /// [the Nomicon](../../nomicon/phantom-data.html).
 ///
-/// # A ghastly note 👻👻👻
+/// # A ghastly note emoji unicode: 1f47bemoji unicode: 1f47bemoji unicode: 1f47b
```

Comment:
🤦 apparently github suggestions don't work with emojis. Do you mind making the change locally?

Reply:
:laughing: Thanks for catching that and your super fast responses. :kissing_heart:
I'll make the changes locally, now (first time, I am pulling the Rust code base on my computer, hui :smiling_face_with_three_hearts: )